### ISSUE TITLE
adjust navbar show distance to 4rem

### ISSUE THREAD

**Issue by LoneWolf4713** [RemyIsCool/userChrome.css#6]
Hello @RemyIsCool, this is actually great, loved this honestly, but I had a problem that the navbar was showing itself below my cursor, I mean, when I tried to reach the tabs, to make searchbar appear, it would appear below my cursor and hence I could neither click nor use it. 
so I changed the navbar appearance distance from 3.5rem to 4rem here: 
```
#navigator-toolbox:is(:hover, :focus-within, :has([open])) {
        #nav-bar {
                top: 4rem;
                opacity: 1;
        }

        #tabs-newtab-button {
                opacity: 1;
        }
}
```
and it works fine now, so this is not much of an issue, but just letting you know, 3.5rem was a problem on my 1920x1080p display! 
Thanks!

**Comment by remyrf** [comment]
Very interesting. Could you please send a screenshot of the issue?

**Comment by dazziedez** [comment]
hey!! Sorry for the mess in my pr......... i'm assuming that the issue here is referring to this:
![firefox_7IWPOFu7jX](https://github.com/user-attachments/assets/89f442c6-abee-41ee-a584-1c1a47223fcd)


I have a 1920x1080p display, i found out about this now because this issue doesn't exist when setting the title bar density to compact
![firefox_ir52QxA3Tv](https://github.com/user-attachments/assets/c8a4f94d-eef4-429c-9fad-4979389bd9ff)


So maybe include that in the readme or implement this change, up to you!! It's 4 am idk i hope i helped..

**Comment by LoneWolf4713** [comment]
hey @RemyIsCool why not, here you go, and yea @dazziedez something like that, I managed to screen record, here:
this is what happens on 3.5rem and see how the navbar appears when I hover over tabs, not when I hover over where it appears, and sometimes, when I lower the mouse from tabs to click on navbar it disappears (as if I had gone outside the area where it appears) and when I change it to 4rem, all this gets resolved

https://github.com/user-attachments/assets/0311aeff-63b4-4bf1-86d6-0c2ba4af3a54

**Comment by remyrf** [comment]
@LoneWolf4713 That's all intended behaviour, except for the navbar jumping around like that. I am working on a fix now.

**Comment by remyrf** [comment]
Fixed in commit [ac87594](https://github.com/RemyIsCool/AnimatedFox/commit/ac8759402b4017b2258c9166b57af9749ac2328d)
